doc/python/wide-form.md

```diff
@@ -0,0 +1,260 @@
+
+fig = px.area(wide_df)
+fig.show()
+
+fig = px.line(wide_df)
+fig.show()
+
+fig = px.scatter(wide_df)
+fig.show()
+```
+
+### Orientation Control When Using Defaults
+
+If you specify neither `x` nor `y`, you can specify whether the Y or X xaxis is assigned to the index with `orientation`.
+
+```python
+import plotly.express as px
+wide_df = px.data.short_track_wide(indexed=True)
+
+fig = px.bar(wide_df, orientation="h")
+fig.show()
+```
+
+### Assigning Columns to Non-Color Arguments
+
+
+In the examples above, the columns of the wide data frame are always assigned to the `color` argument, but this is not a hard constraint. The columns can be assigned to any Plotly Express argument, for example to accomplish faceting, and `color` can be reassigned to any other value. When plotting with a data frame without named indexes, you can reassign the inferred column named `"variable"` and `"value"` to any argument:
+
+```python
+import plotly.express as px
+wide_df = px.data.short_track_wide(indexed=False)
+
+fig = px.bar(wide_df, x="nation", y=["gold", "silver", "bronze"], facet_col="variable", color="nation")
+fig.show()
+```
+
+If using a data frame's named indexes, either explicitly or relying on the defaults, the index references or names must be used:
+
+```python
+import plotly.express as px
+wide_df = px.data.short_track_wide(indexed=True)
+
+fig = px.bar(wide_df, facet_col="medal", color=wide_df.index)
+fig.show()
+```
+
+### Mixed-Form Data
+
+In some cases, a data frame is neither clearly long-form nor wide-form, and we can call this "mixed-form". For example, in the data frame below, if it contained only the `experiment` columns, the data could be described as wide-form, and if it contained only `gender` and `group` it could be described as long-form, but it contains both:
+
+```python
+import plotly.express as px
+mixed_df = px.data.experiment(indexed=True)
+mixed_df.head()
+```
+
+We can visualize just the wide-form portion of the data frame easily with a [violin chart](/python/violin/). As a special note, we'll assign the index, which is the participant ID, to the hover_data, so that hovering over outlier points will identify their row.
+
+```python
+import plotly.express as px
+mixed_df = px.data.experiment(indexed=True)
+
+fig = px.violin(mixed_df, y=["experiment_1", "experiment_2", "experiment_3"], hover_data=[mixed_df.index])
+fig.show()
+```
+
+
+
+
+We can also leverage the long-form portion of the data frame, for example to color by `gender` and facet by `group`:
+
+```python
+import plotly.express as px
+mixed_df = px.data.experiment(indexed=True)
+
+fig = px.violin(mixed_df, y=["experiment_1", "experiment_2", "experiment_3"], 
+                color="gender", facet_col="group", hover_data=[mixed_df.index])
+fig.show()
+```
+
+And of course, we can reassign `variable` to another argument as well, in this case we'll assign it to `x` and facet by the wide variable, and we'll switch to a [box plot](/python/box-plots/) for variety.
```

Comment:
I don't understand this sentence ("we can reassign `variable` to another argument as well, in this case we'll assign it to `x`").

Reply:
yeah that sentence made no sense... fixed!